independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/spi/ServerMessageBodyWriter.java

```diff
@@ -9,7 +9,6 @@
 /**
  * Extension of MessageBodyWriter which can write directly to a Vert.x response
  */
-// FIXME: do we actually need to make it extend MessageBodyWriter?
```

Comment:
We do indeed need to extend `MessageBodyWriter` because otherwise RR doesn't know it's a Writer.

Reply:
Well yes, but it could be a distinct interface.

Reply:
It would save having to implement a useless method.

Reply:
Yeah, we can do a further cleanup (with support in the processor)